internal/pkg/auth/auth_token_handler.go

```diff
@@ -80,23 +81,34 @@ func (a *AuthTokenHandlerImpl) GetCCloudTokens(clientFactory CCloudClientFactory
 }
 
 func (a *AuthTokenHandlerImpl) getCCloudSSOToken(client *ccloudv1.Client, noBrowser bool, email, orgResourceId string) (string, string, error) {
-	userSSO, err := a.getCCloudUserSSO(client, email, orgResourceId)
-	if err != nil {
-		log.CliLogger.Debugf("unable to obtain user SSO info: %v", err)
-		return "", "", errors.Errorf(errors.FailedToObtainedUserSSOErrorMsg, email)
-	}
-	if userSSO == "" {
-		return "", "", errors.Errorf(errors.NonSSOUserErrorMsg, email)
+	isOkta := sso.GetCCloudEnvFromBaseUrl(client.BaseURL) == "fedramp-internal"
+
+	var auth0ConnectionName string
+	if !isOkta {
+		userSSO, err := a.getCCloudUserSSO(client, email, orgResourceId)
+		if err != nil {
+			log.CliLogger.Debugf("unable to obtain user SSO info: %v", err)
+			return "", "", errors.Errorf(errors.FailedToObtainedUserSSOErrorMsg, email)
+		}
+		if userSSO == "" {
+			return "", "", errors.Errorf(errors.NonSSOUserErrorMsg, email)
+		}
+		auth0ConnectionName = userSSO
 	}
 
-	idToken, refreshToken, err := sso.Login(client.BaseURL, noBrowser, userSSO)
+	idToken, refreshToken, err := sso.Login(client.BaseURL, noBrowser, auth0ConnectionName)
```

Comment:
`auth0ConnectionName` would be empty for fedramp?

Reply:
Yes, that's intended! It won't get added to the API call if it's `""`.